src/widgets/helper/ChannelView.cpp

```diff
@@ -130,6 +130,27 @@ ChannelView::ChannelView(BaseWidget *parent)
     this->clickTimer_->setSingleShot(true);
     this->clickTimer_->setInterval(500);
 
+    // TODO(leon): What interval to choose?
+    this->scrollTimer_.setInterval(20);
+    QObject::connect(&this->scrollTimer_, &QTimer::timeout, this, [this] {
+        const qreal delta = this->currentMousePosition_.y() -
+                            this->lastMiddlePressPosition_.y();
+
+        // TODO(leon): How large should the "grace area" be?
```

Comment:
doesn't account for scale, so if you're using 4k then it's only 5 px

Reply:
I believe that this is fixed by 8f2db4d. Can't test though because I don't have a high DPI monitor.